static/templates/recent_pm_row.hbs

```diff
@@ -0,0 +1,49 @@
+<tr id="recent_pm:{{pm_key}}" {{#if unread_count}}class="unread_topic"{{/if}} data-unread-count="{{unread_count}}" data-muted="{{muted}}" data-participated="true">
+    <td class="recent_topic_stream">
+        <div class="flex_container">
+            <div class="left_part recent_topics_focusable">
+                <span class="fa fa-envelope"></span>
+                <a href="{{pm_url}}">Private messages</a>
```

Comment:
All strings should be tagged for translation. 

https://zulip.readthedocs.io/en/latest/translating/internationalization.html